src/ecc/rtl/ecc_reg.rdl

```diff
@@ -66,54 +64,46 @@ addrmap ecc_reg {
+        name = "ECC CTRL";
+        desc = "ECC component control register type definition";
 
         default sw = w;
         default hw = r;
         default resetsignal = reset_b;
-        field {desc = "Control command field"; swwe = ecc_ready; hwclr;} CTRL[2] = 2'b0;
-        field {desc = "Zeroize all internal registers"; singlepulse;} ZEROIZE = 1'b0;
-        field {desc = "Run PCR Signing flow"; swwe = ecc_ready; hwclr;} PCR_SIGN = 1'b0;
+        field {desc = "Control command field: This can be: 
+                [br]             00 for NONE
+                [br]             01 for KEYGEN
+                [br]             10 for SIGNING
+                [br]             11 for VERIFYING"; swwe = ecc_ready; hwclr;} CTRL[2] = 2'b0;
+        field {desc = "Zeroize all internal registers: Zeroize all internal registers after ECC process, to avoid SCA leakage."; singlepulse;} ZEROIZE = 1'b0;
+        field {desc = "Run PCR Signing flow: Run ECC Signing flow to sign PCRs."; swwe = ecc_ready; hwclr;} PCR_SIGN = 1'b0;
     
     } ECC_CTRL @0x00000010;
 
 
 
     /* ---- ECC Component Status ---- */
     reg {
-        name = "ECC component status register type definition";
-        desc = "One 2-bit register including the following flags: 
-                bit #0: READY : ​Indicates if the core is ready to take
-                                a control command and process the block.  
-                bit #1: VALID : ​Indicates if the process is done and the
-                                hash value stored in DIGEST registers is valid. 
-                This register is located at ECC_base_address + 0x0000_0018.";
+        name = "ECC STATUS";
+        desc = "ECC component status register type definition";
 
         default sw = r;
         default hw = w;
         default resetsignal = reset_b;
-        field {desc = "Status ready bit";} READY = 1'b0;
-        field {desc = "Status valid bit";} VALID = 1'b0;
+        field {desc = "Status ready bit: ​Indicates if the core is ready to take
+                                a control command and process the block.";} READY = 1'b0;
+        field {desc = "Status valid bit: ​Indicates if the process is done and the
+                                hash value stored in DIGEST registers is valid.";} VALID = 1'b0;
     
     } ECC_STATUS @0x00000018;
 
 
     /* ---- ECC Component Input SEED ---- */
     reg {
-        name = "ECC component seed register type definition";
-        desc = "12 32-bit registers storing the 384-bit seed for keygen.
-                The seed can be any 384-bit value in [0 : 2^384-1].
-                These registers are located at ECC_base_address + 
-                0x0000_0080 to 0x0000_00AC in big-endian representation.";
+        name = "ECC SEED";
+        desc = "ECC component seed register type definition
+                12 32-bit registers storing the 384-bit seed for keygen.
+                The seed can be any 384-bit value in [0 : 2^384-1].";
```

Comment:
Probably still want to have 'big-endian representation' defined here.

Reply:
added here and all other fields where requried.